templates/unattended-upgrades.j2

```diff
@@ -30,21 +30,35 @@ APT::Periodic::RandomSleep "{{unattended_random_sleep}}";
 
 
 // Overrides for `50unattended-upgrades`. ////////////////////////////////////
-
-// Unattended-Upgrade::Origins-Pattern controls which packages are
-// upgraded.
-Unattended-Upgrade::Origins-Pattern {
-{% if unattended_origins_patterns is defined %}
-  {% for origin in unattended_origins_patterns %}
-    "{{ origin }}";
-  {% endfor %}
-{% else %}
-  {% for origin in __unattended_origins_patterns %}
-   "{{ origin }}";
-  {% endfor %}
+{% if unattended_origins_patterns is defined or __unattended_origins_patterns is defined %}
+  // Unattended-Upgrade::Origins-Pattern controls which packages are
+  // upgraded.
+  Unattended-Upgrade::Origins-Pattern {
+  {% if unattended_origins_patterns is defined %}
+    {% for origin in unattended_origins_patterns %}
```

Comment:
In this first loop that deals with origins-patterns, could we rename `origin` to `pattern` wherever it's used as a jinja variable? I think it would really help with reading the difference between the two sets.

Reply:
Done.